app/ui.js

```diff
@@ -869,6 +872,24 @@ const UI = {
         }
     },
 
+    updateCapabilities() {
+        UI.updatePowerButton();
+        UI.updateAudioButton();
+    },
+
+    updateAudioButton() {
+        if (UI.connected &&
+            UI.rfb.capabilities.audio) {
+            document.getElementById('noVNC_audio_button')
+                .classList.remove("noVNC_hidden");
+            document.getElementById('noVNC_audio_button')
+                .classList.remove("noVNC_selected");
```

Comment:
How often is `updateCapabilities` called? Should we always remove this selected class?

Reply:
at most four times per connection: once after the connection is established (to clear everything), once when the power capabilities are advertised by the server, once when the server says that it supports audio (only once if the user opted in), and once more when the connection is lost (to clean everything again for good measure).

it should be roughly fine to always attempt to remove the selected class since `classList.remove()` is a no-op if the class is not there, and other UI functions follow roughly the same pattern (like `updatePowerButton()`, another funciton that's called from `updateCapabilities()`).

which is a very long way of saying _i guess_ it's fine?

Reply:
if they do this elsewhere, sounds good to me haha!